src/server/migrations/20200608220409_game_instances.js

```diff
@@ -3,7 +3,12 @@
 exports.up = function (knex) {
   return knex.schema.createTable('game_instances', (table) => {
     table.increments('id');
-    table.integer('score');
+    table.integer('fk_game_scores_id');
```

Comment:
As discussed on Sunday, all the foreign keys should be notNullable(), here, it could be to have: 
```

table.integer('fk_game_scores_id').unsigned().notNullable();
table.foreign('fk_game_scores_id').references('id').inTable('game_scores');
```

This applies for all the FK you have in this file.